### DISCUSSION TITLE
How to get the option `--properties-by` available in the `lsblk` commands?

### DISCUSSION THREAD

**Discussion by Ricky-Tigg** [util-linux/util-linux#3195] [Q&A] [answered]
Hello. Elsewhere seen in the _Discussions_ section, as part of an lsblk command, itself as part of the current Git tree, the command option` --properties-by` is not provided in the related package of the latest v. 2.40.2 shipped by the distribution I use. That can either be deduced from the outputs resulting from `man 8 lsblk` and `lsblk --help`. Can this be interpreted as an indication of oversight in [tags](https://github.com/util-linux/util-linux/tags)?.

**Comment by karelzak** [comment, marked as answer]
`lsblk --properties-by`  is available in the `master` branch in the git tree. It has not been released yet, so there is no git tag (or official tarball). It will be released in v2.41. You need to pull the current master branch to play with it.
